public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -295,3 +296,94 @@ export function getStreamSelectorsFromQuery(query: string): string[] {
 
   return labelMatchers;
 }
+
+export function requestSupportsPartitioning(queries: LokiQuery[]) {
+  /*
+   * For now, we would not split when more than 1 query is requested.
+   */
+  if (queries.length > 1) {
```

Comment:
If you have one visible query, and _n_ hidden queries, this check will fail, might want to add:
`queries.filter(query => query.hide).length > 1`

Reply:
Good advice! I'll follow up with this. Thank you.